### DISCUSSION TITLE
Templates and Angular's SSR

### DISCUSSION THREAD

**Discussion by Picoquio** [primefaces/community#1532] [PrimeNG Templates] [answered]
Hi there!

I'm thinking about purchasing one of your templates since I love working with PrimeNG. 

The thing is, we want to develop a website that is server side rendered, using Angular's relatively recent SSR capabilities. My whole frontend dev experience is with client side rendering so this approach is quite new to me, hence I ask: **is there anything I should take into account? Or would the components work just exactly as in client side rendering?**

Thank you.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

All PrimeNG is compatible with SSR, you can implement SSR into your template. It isn't enabled by default, you should configure it.
